src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -223,7 +223,29 @@ public function regenerate($destroy = false, $lifetime = null)
      */
     public function save()
     {
-        session_write_close();
+        try {
+            session_write_close();
+        } catch (\ErrorException $e) {
```

Comment:
Yes, currently the session write fails silently in production.

I updated the code to check the last error message and throw a E_USER_WARNING with the corrected message. However, the warning is not logged by default.

Alternatively, we can throw an Excpetion if the write fails, as it is an error anyway. However, it does break BC.